test/Mono.Linker.Tests.Cases.Expectations/Assertions/UnrecognizedReflectionAccessPatternAttribute.cs

```diff
@@ -16,35 +16,24 @@ namespace Mono.Linker.Tests.Cases.Expectations.Assertions
 		Inherited = false)]
 	public class UnrecognizedReflectionAccessPatternAttribute : BaseExpectedLinkedBehaviorAttribute
 	{
-		public UnrecognizedReflectionAccessPatternAttribute (Type reflectionMethodType, string reflectionMethodName, Type[] reflectionMethodParameters,
-			string message = null,
+		public UnrecognizedReflectionAccessPatternAttribute (
+			Type reflectionMethodType,
+			string reflectionMethodName,
+			Type[] reflectionMethodParameters,
+			object message = null,
+			string messageCode = null,
```

Comment:
Why isn't this `int`?

Reply:
I intentionally made all the warning codes in tests strings:
* It makes the tests seem operate on the visible output (the fact that sometimes we check against the internal number is just an implementation detail)
* It makes the tests closer to the user XP
* (low pri) I still hope that one day we will be able to use non-numerical values